examples/group_vars/setup_vars

```diff
@@ -0,0 +1,107 @@
+# TODO: Make hostname a variable instead of the whole API link
+dr_sites_primary_url: "http://mlipchuk.usersys.redhat.com:8080/ovirt-engine/api"
+dr_sites_primary_username: "admin@internal"
+dr_sites_primary_ca_file: "/home/mlipchuk/ovirt-share/etc/pki/ovirt-engine/ca.pem"
+
+dr_sites_secondary_url: "http://10.35.0.140:8080/ovirt-engine/api"
+dr_sites_secondary_username: "admin@internal"
+dr_sites_secondary_ca_file: "/home/mlipchuk/ovirt-share/etc/pki/ovirt-engine/ca.pem"
+
+dr_import_storage:
```

Comment:
s/dr_import_storage/dr_import_storages

I think plural is better here, to express it's a list.

Reply:
done